tests/filecheck/dialects/frep/assembly_printing.mlir

```diff
@@ -0,0 +1,17 @@
+// RUN: xdsl-opt -t riscv-asm %s | filecheck %s
+
+%i0 = riscv.get_register : () -> !riscv.reg<a0>
+%ft0 = riscv.get_float_register : () -> !riscv.freg<ft0>
+%ft1 = riscv.get_float_register : () -> !riscv.freg<ft1>
+frep.outer %i0, 0, 0 ({
+    %ft2 = riscv.vfadd.s %ft0, %ft1 : (!riscv.freg<ft0>, !riscv.freg<ft1>) -> !riscv.freg<ft2>
```

Comment:
Could you change the example to use regular float operation instead of a vector one?
Seems a bit less specfic?